gdal/src/main/scala/geotrellis/gdal/GDALReader.scala

```diff
@@ -77,12 +78,17 @@ class GDALReader(val dataset: Dataset) {
     val typeSizeInBytes = gdal.GetDataTypeSize(bufferType) / 8
     val bufferSize = bandCount * pixelCount * typeSizeInBytes
 
-    val cellType = GDALUtils.dataTypeToCellType(
-      datatype = bufferType,
-      noDataValue = noDataValue,
-      typeSizeInBits = Some(typeSizeInBits),
-      minMaxValues = minMax
-    )
+    val cellType =
+      targetCellType match {
+        case Some(ct) => println(s"\n\nI matched a targetCellType: ${ct}"); ct
```

Comment:
let's remove this println and replace the code with 

```scala
targetCellType.getOrElse(GDALUtils.dataTypeToCellType(
  datatype = bufferType,
  noDataValue = noDataValue,
  typeSizeInBits = Some(typeSizeInBits),
  minMaxValues = minMax
))
```

Reply:
:+1: